tmt/base.py

```diff
@@ -1553,6 +1553,7 @@ def __init__(
                 self._initialize_worktree()
 
             self._initialize_data_directory()
+            self._plan_environment: EnvironmentType = {}
```

Comment:
I'd recommend moving somewhere on line 1566, after `self._environment` initialization: the attribute may exist all the time, it'd just remain empty, making the reasoning about it easier.

The name, the name again! :( I still don't have a better name, but I'd like to see some uniformity: there's `_plan_environment`, `env_file`, and `_source_plan_env_file`. How about renaming the latter two to follow the example of the first one, and stick to "plan_environment": `_plan_environment` to hold the loaded envvars, `plan_environment_file` property providing the path, and `_source_plan_environment_file` populating `_plan_environment` from the said file. When one starts looking for "plan environment", the `plan_environment` substring would point to all bits taking care of its processing.

Reply:
Fixed in e70b67e.